db/sg_replicate_cfg.go

```diff
@@ -252,6 +264,28 @@ func (rc *ReplicationConfig) Equals(compareToCfg *ReplicationConfig) (bool, erro
 	return bytes.Equal(currentBytes, compareToBytes), nil
 }
 
+// Redact returns the ReplicationCfg with auth credentials of the remote database
+// redacted. Credentials will be redacted from replication config and remote URL.
+func (r *ReplicationCfg) Redact() (config *ReplicationCfg) {
```

Comment:
This can be applied on ReplicationConfig instead of cfg. There's no additional information we need to redact in the cfg wrapper.

```suggestion
func (r *ReplicationConfig) Redact() (config *ReplicationConfig) {
```

Reply:
Defined Redact method on ReplicationConfig since there's no additional information we need to redact in the cfg wrapper.